core/src/main/java/org/infinispan/configuration/global/UncleanShutdownAction.java

```diff
@@ -0,0 +1,22 @@
+package org.infinispan.configuration.global;
+
+/**
+ * Defines the action taken when a dangling lock file is found in the persistent global state, signifying an
+ * unclean shutdown of the node (usually because of a crash or an external termination).
+ *
+ * @since 15.0
+ */
+public enum UncleanShutdownAction {
+   /**
+    * Ignores the presence of a dangling lock file in the persistent global state.
+    */
+   NONE,
```

Comment:
isn't IGNORE a better word to describe instead of NONE?

Reply:
Yes. I've changed it